ctypesgen/__main__.py

```diff
@@ -29,19 +29,19 @@ def find_names_in_modules(modules):
 
 def main(givenargs=None):
     # TODO(geisserml) In the future, convert action="append" to nargs="*" - that's nicer to use
-    
```

Comment:
Ideally these changes would go into the other currently unmerged commits (that recently introduced these lines). Using git commit --amend and --rebase. That way the git blame/history points to the logical commit that introduced these changes.